dcc-portal-ui/app/scripts/downloader/js/request.js

```diff
@@ -136,8 +136,10 @@
 
         $scope.params.processing = false;
       },function (error) {
-        $scope.params.processing = false;
-        $scope.params.downloadEnabled = false;
+        if(error.status === 503){
+          $scope.params.processing = false;
+          $scope.params.downloadEnabled = false;
+        }
```

Comment:
@cheapsteak What's the best way to trigger the global error handler for restangular if not a `503` above? `throw error`?